Dockerfile

```diff
@@ -1,32 +1,62 @@
 #
 # Dockerfile for Storm Mesos framework
 #
-FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
-MAINTAINER Timothy Chen <tnachen@gmail.com>
-
-ENV DEBIAN_FRONTEND noninteractive
-
-# export environment
-ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
-
-ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
-
-ARG MESOS_RELEASE=0.27.0
-ARG STORM_RELEASE=0.9.6
-ARG MIRROR=http://www.gtlib.gatech.edu/pub
+FROM ubuntu:14.04
+MAINTAINER Salimane Adjao Moustapha <me@salimane.com>
+
+# Add mesosphere, openjdk 8 repo
+# install jdk 7 and 8
+RUN DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]') && \
+	CODENAME=$(lsb_release -cs) && \
+	echo "deb http://repos.mesosphere.com/${DISTRO} ${CODENAME} main" > /etc/apt/sources.list.d/mesosphere.list && \
+	apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
+	apt-get -y install software-properties-common && \
+	add-apt-repository ppa:openjdk-r/ppa && \
+	apt-get -y update && \
+	apt-get -y install openjdk-7-jdk openjdk-8-jdk && \
```

Comment:
How much extra space are we adding by having both JDK versions in our docker images?  I'm not sure that's a real concern, but if we are experiencing pain due to too-large docker images this would seem an obvious candidate for optimization.  We can leave it be for now, I'm mostly just curious.